### PR TITLE
Migrate to Ballerina Swan Lake Preview1

### PR DESCRIPTION
**Pull request by MadhukaHarith92**

### CODE REVIEW

File: redis-utils/src/main/java/org/ballerinalang/redis/RedisDataSource.java [hunk 1]
```diff
@@ -184,22 +186,25 @@ private void setRedisClusterCommands(List<ServerAddress> serverAddresses, MapVal
...
 
         if (database != -1) {
             builder.withDatabase(database);
         }
         if (connectionTimeout != -1) {
             builder.withTimeout(Duration.ofMillis(connectionTimeout));
         }
-        if (!clientName.isEmpty()) {
-            builder.withClientName(clientName);
+        if (!clientName.toString().equals("")) {
```
Comment (daneshk) [inline redis-utils/src/main/java/org/ballerinalang/redis/RedisDataSource.java:206]:
```suggestion
        if (clientName.length() != 0) {
```

File: redis-utils/src/main/java/org/ballerinalang/redis/RedisDataSource.java [hunk 2]
```diff
@@ -184,22 +186,25 @@ private void setRedisClusterCommands(List<ServerAddress> serverAddresses, MapVal
...
+            builder.withClientName(clientName.toString());
```
Comment (daneshk) [inline redis-utils/src/main/java/org/ballerinalang/redis/RedisDataSource.java:207]:
```suggestion
            builder.withClientName(clientName.getValue());
```

File: redis/src/redis/endpoint.bal [hunk 3]
```diff
@@ -23,7 +23,7 @@ public type Client client object {
     handle datasource;
 
     # Gets called when the endpoint is being initialized during the module initialization.
-    public function __init(ClientEndpointConfiguration config) returns Error? {
+    public function init(ClientEndpointConfiguration config) returns error? {
```
Comment (kalaiyarasiganeshalingam) [inline redis/src/redis/endpoint.bal:26]: We are using a distinct error in most of the connectors. I think, If we haven't any specific reason to change the `Error` to `error`, we can keep this.

**Comment by daneshk** [comment]
@MadhukaHarith92 Shall we move redis connector to `ballerinax`. related issue #58

**Comment by MadhukaHarith92** [comment]
> @MadhukaHarith92 Shall we move redis connector to `ballerinax`. related issue #58

Ack.

File: redis/src/redis/endpoint.bal (hunk 3 above)
Comment (MadhukaHarith92) [reply redis/src/redis/endpoint.bal]: Reverted back in f8eade9

**Pull request merged**